circuits/cpp/src/aztec3/circuits/kernel/public/.test.cpp

```diff
@@ -60,12 +60,12 @@ template <size_t SIZE>
 std::array<NT::fr, SIZE> array_of_values(NT::uint32& count, NT::uint32 num_values_required = SIZE)
 {
     std::array<NT::fr, SIZE> values;
-    size_t i = 0;
-    for (; i < num_values_required; i++) {
-        values[i] = ++count;
+    size_t ind = 0;
+    for (; ind < num_values_required; ind++) {
+        values[ind] = ++count;
     }
-    for (; i < SIZE; i++) {
-        values[i] = 0;
+    for (; ind < SIZE; ind++) {
+        values[ind] = 0;
```

Comment:
You can ignore the variable length warning for loop indices IMO. Just use your best judgement.